rocketpy/environment/environment.py

```diff
@@ -3083,7 +2393,7 @@ def calculate_density_profile(self):
         >>> env = Environment()
         >>> env.calculate_density_profile()
         >>> float(env.density(1000))
-        1.1116193933422585
+        1.1115112430077818
         """
```

Comment:
Why did this change?

Reply:
I honestly don`t know. The difference was so small that I ignored it for a moment.
However, I'm curious now.

Maybe it was related to the discretization of barometric pressure somehow.